src/components/form-draft/testing.vue

```diff
@@ -167,11 +164,11 @@ export default {
     // This text will be shown in ODK Collect when testing a Draft Form. {name}
     // is the title of the Draft Form.
     "collectProjectName": "[Draft] {name}",
-    "datasetsPreview": {
+    "entitiesTesting": {
+      "title": "This Form can create Entities",
       "body": [
-        "This Form can add Entities to a Dataset.",
-        "In this preview release of Datasets, you must publish your Form to see the Dataset update. In a future release of Central, you will be able to test Dataset functionality in a Draft state.",
-        "For now, we recommend publishing your Dataset-related Forms, and not approving any submissions (and therefore not create any unwanted entities) until you are sure the Form is ready."
+        "Entities are only created for published Forms. In a future release of Central, you will be able to test Entity functionality in a Draft state.",
```

Comment:
> Entities are only created for published Forms.

Submissions to a draft of a published form will create entities right now. Not sure about submissions to a draft form that's never been published. It sounds like draft submissions should maybe never create entities? Here's an example of a draft submission to a published form:

https://staging.getodk.cloud/#/projects/27/forms/trees_registration_demo/draft/testing
https://staging.getodk.cloud/#/projects/27/datasets/trees/entities/576ed88d-9350-4ed1-a295-b2a26a935a6b

Reply:
I guess this wasn't possible before, because you can't approve a draft submission. But now that we have create on receipt, this is a possibility.

Reply:
That definitely looks like a bug.

Reply:
Filed an issue here: getodk/central-backend#880